docs/checks/roundaboutMissingTagCheck.md

```diff
@@ -0,0 +1,39 @@
+# Roundabout Missing Tag Check
+
+#### Description
+This check identifies closed navigable ways https://wiki.openstreetmap.org/wiki/Item:Q4669 
+that have a round shape and intersecting with at least two navigable ways.    
+
+#### Live Example
+Roundabout without `junction=roundabout` tag.
+1) [id:592687817](https://www.openstreetmap.org/way/592687817) 
+2) [id:548529232](https://www.openstreetmap.org/way/548529232)
+3) [id:150664424](https://www.openstreetmap.org/way/150664424)
+4) [id:287957529](https://www.openstreetmap.org/way/287957529)
+
+#### Code Review
+
+In [Atlas](https://github.com/osmlab/atlas), OSM elements are represented as Edges, Points, Lines, 
+Nodes & Relations; in our case, we’re working with [Edges](https://github.com/osmlab/atlas/blob/dev/src/main/java/org/openstreetmap/atlas/geography/atlas/items/Edge.java).
+In OpenStreetMap, roundabouts are [Ways](https://wiki.openstreetmap.org/wiki/Way) classified with
+the `junction=roundabout` tag. We’ll identify ways that are roundabouts without `junction=roundabout` tag.
+
+Our first goal is to validate the incoming Atlas Object.
+* Must be a first section of valid master Edge
+* Must have not already been flagged
+* Must be car navigable
+* Must have not `junction` and `area` tag
+* Must be a closed way [OSM-wiki:Closed Way](https://wiki.openstreetmap.org/wiki/Item:Q4669)
+* Must intersect at minimum with two navigable ways 
+
+After the preliminary filtering of features we need to rebuild original OSM way geometry.
```

Comment:
If we're making sure a valid edge is the first section of a way to ensure that the way has more than 2 edges, we could note that here

Reply:
its noted: 
* Must be a first section of valid master Edge
* Must intersect at minimum with two navigable ways

Reply:
Right -- i was more referring to the "why" we need to ensure a valid edge is the first section of the edge, as opposed to some other part of the edge. I might be alone in thinking that level of documentation is necessary so up to you!